zerver/openapi/zulip.yaml

```diff
@@ -1383,36 +1383,29 @@ paths:
                                 streams:
                                   type: array
                                   description: |
-                                    Array of objects, each containing
-                                    details about a channel that was deleted.
+                                    Array of objects, each containing stream_id
```

Comment:
We should mention that this field is deprecated. And can phrase this to be a complete sentence like we had before.

Reply:
e.g. "Deprecated and will be removed in a future release" ?

Reply:
You can look at other examples for deprecated fields.